app/src/main/AndroidManifest.xml

```diff
@@ -15,12 +15,8 @@
     <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
     <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
     <uses-permission android:name="android.permission.MANAGE_EXTERNAL_STORAGE" />
-    <uses-permission
-        android:name="android.permission.READ_CALL_LOG"
-        tools:node="remove" />
-    <uses-permission
-        android:name="android.permission.READ_CONTACTS"
-        tools:node="remove" />
+    <uses-permission android:name="android.permission.READ_CALL_LOG"/>
```

Comment:
why we have this permission again?

Reply:
I'm not sure, but from what I checked it seems that access to the phone app is needed to read the IMEI of the phone, and it was a feature. I don't think we need it currently. Should I remove it?